docs/2.utils/98.advanced.md

```diff
@@ -151,6 +151,28 @@ If the incoming request is a CORS preflight request, it will append the CORS pre
 
 If return value is `true`, the request is handled and no further action is needed.
 
+**Example:**
+
+```ts
+const app = createApp();
+const router = createRouter();
+router.use('/',
+  defineEventHandler(async (event) => {
+      const didHandleCors = handleCors(event, {
+        origin: '*',
+        preflight: {
+         statusCode: 204,
+        },
+     methods: '*',
+   });
+   if (didHandleCors) {
+     return;
+   }
+   // Your code here
```

Comment:
@jolo-dev this should go inside the callback for `defineEventHandler`, shouldn't it?

Reply:
@chadxz 

You are right, and it has been already fixed by the package author:

https://github.com/unjs/h3/blob/5599d61ae5b1e32d9bfa222ccb90bee475ce675e/src/utils/cors.ts#L65-L98

Reply:
@nozomuikuta thanks for your reply ❤️ i noticed some odd indentation in the docs here: https://h3.unjs.io/utils/advanced#handlecorsevent-options and had previously been looking at this PR, so came here and it was slightly different and wrong.

The code sample you linked to also looks wrong and is also different than the deployed docs. This is what I see at the moment:

<img width="464" alt="Screenshot 2025-01-19 at 9 28 25 AM" src="https://github.com/user-attachments/assets/de94a9a6-8359-4465-848c-1c6ddaae0a02" />

Anyway, I'll take a look at main and see if there's anything that still needs fixing that I can submit a PR for.

Reply:
https://github.com/unjs/h3/pull/958